server/routes/articles.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const router = express.Router();
+const Article = require('../models/article');
+
+//put 1 article
+router.post('/', async (req, res) => {
+  const article = new Article({
+    title: req.body.title,
+    description: req.body.description,
+    author: req.body.author,
+    tags: req.body.tags,
+  });
+  try {
+    const newArticle = await article.save();
+    //201 cause its sucess
+    res.status(201).json(newArticle);
+  } catch (arr) {
+    res.status(400).json({ message: err.message });
+  }
+});
+
+//get all articles
+router.get('/', async (req, res) => {
+  try {
+    //todo--> add more serch and filter options
+    const articles = await Article.find();
+
+    //since for now there is no search option
+    //our req is always sucess so 201
+    res.status(201).json(articles);
+  } catch (err) {
+    //500 for any internal error i.e my fault
+    res.status(500).json({ message: err.message });
+  }
+});
```

Comment:
Only function calls should be there, Logic should be removed 
<img width="1209" alt="Screenshot 2020-04-25 at 6 59 57 PM" src="https://user-images.githubusercontent.com/21286134/80281228-5ce1d600-8727-11ea-97cf-9acaac628126.png">